README.md

```diff
@@ -599,7 +599,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Euhid Aman](https://github.com/euhidaman)
   - [Evandro Rodrigues](https://github.com/evnrodr)
   - [Ezinne Anne Emilia](https://github.com/ezinneanne)
-  - [Edson Sooraj Dsouza](https://github.com/edson-sooraj)
+  
```

Comment:
Remove this line number 602

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Hey, @RakeshSangem removed the extra space.